walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -15,9 +15,102 @@
 package com.google.sps;
 
 import java.util.Collection;
+import java.util.Collections;
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+
+    // first iterate through and combine events that overlap
+    // subtract combined events calendar from whole day
+    // remove time ranges that are too short 
 
 public final class FindMeetingQuery {
   public Collection<TimeRange> query(Collection<Event> events, MeetingRequest request) {
-    throw new UnsupportedOperationException("TODO: Implement this method.");
+        List<TimeRange> avaliableRanges = new ArrayList<TimeRange>();
```

Comment:
It's unnecessary to specify the type: `new ArrayList<>()`

also typo: availableRanges

Reply:
Thanks